packages/arb-token-bridge-ui/src/util/SentryUtils.ts

```diff
@@ -0,0 +1,33 @@
+import * as Sentry from '@sentry/react'
+
+export function captureSentryErrorWithExtraData({
+  error,
+  originFunction,
+  erc20ParentAddress,
+  additionalData
+}: {
+  error: unknown
+  originFunction: string
+  erc20ParentAddress?: string
```

Comment:
Why can't this also be a part of additional data only?  `additionalData` is pretty flexible IMO.
Infact we can also adjust `originFunction` within `additionalData`, no? 

The function signature will be quite easy - `captureSentryErrorWithExtraData(error, additionalData)`

Reply:
i can move `erc20ParentAddress` into `additionalData`, but i don't think `originFunction` should be moved as it should be present in every usage